Daytime-Operations/Auxiliary-Telescope/AuxTel-E-Stop-Procedure/E-Stop-Procecure.rst

```diff
@@ -0,0 +1,113 @@
+    :name: e-stop-dome-control-box
+
+    Screenshot of the e-stop system located in the Dome Control Box.
+
+Additionally, a third emergency stop button is available upstairs in the control box used for manual movement of the dome components, 
+as illustrated in the accompanying figure. 
+It is worth noting, however, that this control box can only be used to initiate the emergency stop and cannot be used to release it, 
+as it does not have a ``Start`` button on it.
+
+.. figure:: ./_static/e-stop-up-stairs.jpeg
+    :name: e-stop-up-stairs
+
+    Screenshot of the e-stop system located upstairs.
+
+In the future, an emergency stop will also be added to the main telescope control room. 
+For now, the emergency e-stop system available in there will only work for the main telescope components.
+
+.. _E-Stop-Procedure-How-Engage-Disengage-It:
+
+How to Engage/Disengage It
+==========================
+
+**Engaging** the emergency stop system is a straightforward process that is both efficient and reliable. 
+To trigger it, simply **pull the Emergency Stop button**. To **release it**, simply **turn the red button clockwise** a quarter or so
+and then **pull the Start** button.
+
+.. figure:: ./_static/e-stop-buttons.jpeg
+    :name: e-stop-buttons
+
+    Screenshot the e-stop ``Emergency Stop`` and ``Start`` buttons.
+
+**Verifying** that the emergency stop system was properly engaged is also easy and straightforward. 
+You just have to look at the computer located on the first floor of the AuxTel building. 
+If the emergency stop was correctly triggered, you will see the indicator light up red, as depicted in the accompanying figure. 
+Conversely, when the emergency stop is released, the red light will be turned off, indicating that the system has been successfully restarted.
+From that points onwards, the telescope will be released to move, either local or remotely.
+
+.. figure:: ./_static/e-stop-indicator-red.jpeg
+    :name: e-stop-indicator-red
+
+    Screenshot the e-stop status indicator. If a red light is turned on in the ``Emergency Stop`` indicator,
+    the e-stop has been activated correctly.
+
+.. note:: 
+    The system is smart enough to allow an independent operation, meaning that you can disengage the emergency stop from any location, 
+    regardless of where it was initially triggered. This system does not cut power to the instruments themselves, which means you still
+    can take images while the e-stop is engaged.  
+
+.. figure:: ./_static/e-stop-eletronics.jpeg
+    :name: e-stop-electronics
+
+    The Pilz controller inside the ``AT Control Cabinet`` located in the first floor. 
+    All three lights illuminated indicates that the E-stops are properly deactivated.
+
+
+If you open the ``AT Control Cabinet``, you may see what is shown in the figure above. 
+The three LEDs brightly illuminated indicate that the system is working correctly and the AuxTel telescope is released to me moved.
```

Comment:
"to me moved" . -- > "to be moved"
or 
"the AuxTel telescope is released to me moved." -- > "the AuxTel is ready to move".